<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1241</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CellAnalyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="input_picture_label">
      <property name="minimumSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="save_btn">
        <property name="text">
         <string>Uložiť</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="load_btn">
      <property name="text">
       <string>Načítať</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="output_picture_label">
      <property name="minimumSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>512</width>
        <height>512</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Zadaj hodnotu (9,12,15,18,21,24)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Zadaj hodnotu (red, green, blue)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="btn_exit">
        <property name="text">
         <string>Ukonči</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="btn_vertical">
        <property name="text">
         <string>Preklop vertikálne</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="btn_horizont">
        <property name="text">
         <string>Preklop horizontálne</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="btn_channel">
        <property name="text">
         <string>Vyber kanál</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btn_depth">
        <property name="text">
         <string>Zmeň farebnú presnosť</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string></string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="depth_edit_line"/>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="channel_edit_line"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
